src/universal/modules/meeting/components/MeetingLobby/MeetingLobby.js

```diff
@@ -16,7 +16,7 @@ const createStartMeetingHandler = (members) => () => {
   if (!self) {
     throw new Error('You are not a member! How can that be?');
   }
-  const firstFacilitator = members.find(member => member.isFacilitator);
+  const firstFacilitator = members.find((m) => m.isFacilitator && m.isConnected);
```

Comment:
yep, `isFacilitator` is a property of teamMember. We'll be able to adjust that in team settings. This looks confusing. lemme rewrite.